crates/ndc-query-plan/src/plan_for_query_request/query_plan_state.rs

```diff
@@ -159,18 +170,7 @@ impl<T: QueryContext> QueryPlanState<'_, T> {
         variable_name: &ndc::VariableName,
         expected_type: Type<T::ScalarType>,
     ) {
-        self.register_variable_use_helper(variable_name, Some(expected_type))
-    }
-
-    pub fn register_variable_use_of_unknown_type(&mut self, variable_name: &ndc::VariableName) {
-        self.register_variable_use_helper(variable_name, None)
-    }
-
-    fn register_variable_use_helper(
-        &mut self,
-        variable_name: &ndc::VariableName,
-        expected_type: Option<Type<T::ScalarType>>,
-    ) {
+        // self.register_variable_use_helper(variable_name, Some(expected_type))
```

Comment:
Why is this commented out?

Reply:
Oh good catch! There was a point where I had split out a helper function to cover some of that function's logic, but then I ended up keeping everything in one function.